tools/generate_docker_image_names.py

```diff
@@ -0,0 +1,112 @@
+"""
+Dockerリポジトリ名、バージョン文字列、カンマ区切りのプレフィックスを受け取り、
+バージョン文字列付きのDockerイメージ名を改行区切りで標準出力に出力する
+
+例1
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "REPO" \
+  --version "VER" \
+  --prefix ",A,B"
+REPO:VER
+REPO:A-VER
+REPO:B-VER
+
+例2
+$ python3 ./tools/generate_docker_image_names.py \
+  --repository "REPO" \
+  --version "VER" \
+  --prefix ""
+REPO:VER
```

Comment:
（かなり細かいけど）

何番目の例かは省けそう
```suggestion
例
$ python3 ./tools/generate_docker_image_names.py \
  --repository "REPO" \
  --version "VER" \
  --prefix ",A,B"
REPO:VER
REPO:A-VER
REPO:B-VER

$ python3 ./tools/generate_docker_image_names.py \
  --repository "REPO" \
  --version "VER" \
  --prefix ""
REPO:VER
```